lib/mongoid_session_store/store.rb

```diff
@@ -0,0 +1,69 @@
+require 'action_dispatch/middleware/session/abstract_store'
+require 'mongoid_session_store/session'
+
+module MongoidSessionStore
+  class Store < ActionDispatch::Session::AbstractStore
+    SESSION_RECORD_KEY = 'rack.session.record'.freeze
+    SESSION_OPTIONS_KEY = Rack::Session::Abstract::ENV_SESSION_OPTIONS_KEY
+
+  private
+
+    def generate_sid
+      BSON::ObjectId.new.to_s
+    end
```

Comment:
This is the main change here, we favour `BSON::ObjectId` instead of using `SecureRandom`.